docs/integrations/sources/google-drive.md

```diff
@@ -0,0 +1,247 @@
+# Google Drive
+
+This page contains the setup guide and reference information for the Google Drive source connector.
+
+:::info
+The Google Drive source connector pulls data from a single folder in Google Drive. Subfolders are recursively included in the sync. All files in the specified folder and all root folders will be considered.
```

Comment:
```suggestion
The Google Drive source connector pulls data from a single folder in Google Drive. Subfolders are recursively included in the sync if specified in the config. All files in the specified folder and all root folders will be considered.
```

Reply:
I was just suggesting this because the glob patterns will determine whether we include subfolders in the sync. But my suggestion is vague, maybe something like "if specified by the glob patterns that the user configures for the sync" is better.